docs/examples/use_cases/tensorflow/efficientdet/eval.py

```diff
@@ -0,0 +1,75 @@
+import multiprocessing
+from absl import logging
+import numpy as np
+import tensorflow as tf
+import re
+import os
+
+import hparams_config
+import utils
+from model import efficientdet_net
+from pipeline.dali import fn_pipeline
+
+
+def run_eval(args):
+    logging.set_verbosity(logging.WARNING)
+
```

Comment:
Don't `eval.py` and `train.py` have a few lines of duplicated code? Would it make sense to extract it?

Reply:
I extracted duplicated code where we prepare GPUs (set up memory_growth and soft_device_placement).
When it comes to the marked part of the code the only part I think is possible to extract is
```
    config = hparams_config.get_efficientdet_config(args.model_name)
    config.override(args.hparams, allow_new_keys=True)
    config.image_size = utils.parse_image_size(config.image_size)
```
but I don't think it makes sense to do that (it'd be something like get_config and we have already get_efficientdet_config and I think that would be too confusing).